cqc/MessageHandler.py

```diff
@@ -274,6 +321,7 @@ def _process_command(self, cqc_header, length, data, is_locked=False):
                 logging.debug("CQC %s: Read XTRA Header: %s", self.name, xtra.printable())
 
             # Run this command
+            print(cmd.printable())
```

Comment:
Is this print statement here intentional?

Reply:
That shouldn't be there. I removed it.